src/navigation/dispatch/useAllTasks.js

```diff
@@ -62,5 +53,7 @@ export function useLoadUnassignedTasksInfo(date) {
       refetchUnassignedTasks();
       refetchTaskLists();
     },
```

Comment:
`isFetching: isLoadingTaskLists || isLoadingUnassignedTasks || isLoadingCourierUsers,`

Notice that `isLoading` is different from `isFetching`

> isLoading - When true, indicates that the query is currently loading for the first time, and has no data yet. This will be true for the first request fired off, but not for subsequent requests.

> isFetching - When true, indicates that the query is currently fetching, but might have data from an earlier request. This will be true for both the first request fired off, as well as subsequent requests.

https://redux-toolkit.js.org/rtk-query/usage/queries#frequently-used-query-hook-return-values

Let's try not to mix them up

Reply:
Right! That mix was solved at commit https://github.com/coopcycle/coopcycle-app/pull/1958/commits/d1a9b7602eb592869d962d41ce98f70ca496fd3b